integration/mochaTests/README.md

```diff
@@ -0,0 +1,131 @@
+
+If you have an older version of driver installed then you will need to have an older version of browser on your machine as well. Generally, it is recommended to do local testing on latest version. If you need to test on older version, you can run a browser compatibility test with saucelabs.
+
+Sample command to run an integration test locally:
+
+```bash
+npm run test -- --test-name AudioTest --host local --test-type integration-test
+```
+
+Browser compatiblity tests will run across a variety of browser and OS combinations so it is recommended to use SauceLabs for them.
+
+### Running Browser Compatibility Tests on SauceLabs
+To run the test on SauceLabs, the username and access key will be required for authentication. Update the following commands with the credentials and add the credentials to environment variables.
+```bash
+export SAUCE_USERNAME=<Sauce Labs account username>
+```
+```bash
+export SAUCE_ACCESS_KEY=<Sauce Labs access key>
+```
+
+SauceLabs will open a browser and load a test url. You need to have a serverless deployment of Chime JS SDK demo. You can set the demo url as an environment variable with the following command:
+```bash
+export TEST_URL=<JS SDK Demo URL>
+```
+
+Following command can be used to run a browser compatibility tests with default settings on SauceLabs:
+
+```bash
+npm run test -- --test-name AudioTest --host saucelabs --test-type browser-compatibility
+```
+
+There are scenarios where a test might not be compatible with one of the browsers or OS. In that case, user can provide a custom config with updated clients array. `sample_test.config.json` is a sample test config already provided. 
+Following command can be used to run a browser compatibility test with custom config:
+
+```bash
+npm run test -- --test-name AudioTest --host saucelabs --test-type browser-compatibility --config browserCompatibilityTest/desktop/sample_test.config.json
+```
+
+### Running Browser Compatibility Tests on DeviceFarm
+To run the tests on DeviceFarm, you will need an AWS account. Once you have an AWS account, you will need to setup two environment variables that will allow the AWS SDK to authenticate to your AWS account.
+```bash
+export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+```
+
+You will need to create a DeviceFarm project in the AWS account that you are planning to use. There are two types of devicefarm projects that can be created: `Mobile Device Testing` and `Desktop Browser Testing`. For this section, we will focus mainly on desktop browser testing.
+After the project is created you will need to setup a `PROJECT_ARN` as an environment variable. Project arn is used by the device farm API to identify the project to create test sessions inside.
```

Comment:
- `is created you will` => `is created, you will` (add a comma)
- `setup` => set up`
- `The project ARN` (uppercase ARN)
- `device farm` => `Device Farm`

Or use an active voice.

> Device Farm APIs use the project ARN to identify the project and create test sessions inside.